libcodechecker/storage/report_server.py

```diff
@@ -45,104 +45,6 @@ class CheckerReportHandler(object):
     information to the database.
     """
 
-    def __sequence_deleter(self, table, first_id):
```

Comment:
The bug path representation also changes: instead of linked list we are ordering by an enumeration column, so no sequence deleter is needed.